File: parte001/ex01_area_rectangulo.cs
namespace parte001
{
    class ex01_area_rectangulo
    {
        static void Main(string[] args)
        {
            double baseRectangulo, alturaRectangulo, areaRectangulo, perimetro;

            Console.Write("Introduzca la base del rectángulo: ");
            baseRectangulo = Convert.ToDouble(Console.ReadLine());

            Console.Write("Introduzca la altura del rectángulo: ");
            alturaRectangulo = Convert.ToDouble(Console.ReadLine());

            areaRectangulo = baseRectangulo * alturaRectangulo;

            Console.WriteLine("El área del rectángulo es: " + areaRectangulo);

            perimetro = (baseRectangulo + alturaRectangulo) * 2;

            Console.WriteLine("El perímetro del rectángulo es: " + perimetro);
        }
    }
}

File: parte001/ex01_area_rectangulo.py
# Hacer  un  algoritmo  que,  dados  los  dos  lados  diferentes  de  un  rectángulo,  encuentre  el  perímetro y el área del mismo.

lado_1 = float(input("Ingrese el primer lado del rectángulo: "))
lado_2 = float(input("Ingrese el segundo lado del rectángulo: "))

perimetro = 2 * (lado_1 + lado_2)

area = lado_1 * lado_2

print("El perímetro del rectángulo es: ", perimetro)
print("El área del rectángulo es: ", area)
